src/main/java/com/aws/greengrass/util/orchestration/ProcdUtils.java

```diff
@@ -55,8 +55,10 @@ public boolean setupSystemService(KernelAlternatives kernelAlternatives) {
             // configured in "/bin/sh". So we launch this service through System V style.
             SystemServiceUtils.runCommand(logger, LOG_EVENT_NAME,SERVICE_CONFIG_FILE_PATH + " reload", false);
             SystemServiceUtils.runCommand(logger, LOG_EVENT_NAME,SERVICE_CONFIG_FILE_PATH + " stop", false);
-            SystemServiceUtils.runCommand(logger, LOG_EVENT_NAME,SERVICE_CONFIG_FILE_PATH + " enable", false);
-            SystemServiceUtils.runCommand(logger, LOG_EVENT_NAME,SERVICE_CONFIG_FILE_PATH + " start", false);
+            if (start) {
+                SystemServiceUtils.runCommand(logger, LOG_EVENT_NAME,SERVICE_CONFIG_FILE_PATH + " enable", false);
```

Comment:
service should always be enabled, that way it will start on the next reboot.

Reply:
On Windows the service is always enabled, no additional command.

Reply:
I understand that point of view, yes. Though I think that from a documentation and expectation perspective it would make more sense to have it enabled, but not started.

Reply:
Ok yeah, if windows is effectively enabled but not started then I think it definitely makes sense to be consistent.